frontend/pages/race_analysis.py

```
@@ -0,0 +1,22 @@
+# Import dependencies
+from pages.frontend_sections.running_race_analysis import render_running_race_analysis_section
+from streamlit_components.ui_components import configure_page_config
+from functions.data_functions import Variables
+import streamlit as st
+
+# Set page config
+configure_page_config(repository_name='strava-ui-streamlit',
+                      page_icon='🏃‍♂️')
+
+# Collect codebase variables
+vars = Variables()
```

Ruff (0.15.6)

[error] Variable `vars` is shadowing a Python builtin

(A001)